src/apps/AppCenter/AppCenter.js

```diff
@@ -174,6 +184,12 @@ class AppCenter extends React.Component {
                     items={navigationItems}
                     onBack={this.handleCloseRepo}
                   />
+
+                  {canUpgradeOrg && (
+                    <UpgradeButton mode="strong" onClick={onUpgradeAll}>
+                      Upgrade all
+                    </UpgradeButton>
+                  )}
```

Comment:
Need to double check with @dizzypaty, but not sure if we should always show this button (or hide it when you're in an app's details page).

Reply:
See https://github.com/aragonone/product/issues/90#issuecomment-493525663 (also includes assets for the responsive version).